test/CMakeLists.txt

```diff
@@ -150,83 +150,25 @@ function(add_sanitize_test TEST_SOURCE)
     endforeach()
 endfunction()
 
-# All the tests are manually sorted in descending order of durations taken from
-# Jenkins, "Full long tests"/"HIP Release All". This is to exploit ctest's 
-# parallelizm as much as possible. Before this change, there were some long
-# jobs at the end, utilizing only one CPU core. This order can potentially
-# save up to ~23 minutes (20%) of total time of "HIP Release All".
-#
-# We use two lists, LONG_TESTS and SHORT_TESTS to implement the sorting
-# mentioned above. If you would like to add a new test, insert it into the
-# LONG_TESTS if it takes more than 800 seconds, or to SHORT_TESTS otherwise.
-# Please notice that each list is also sorted and it is highly recommended
-# to keep this sorting when adding new tests.
-
-set( LONG_TESTS
-    pooling2d.cpp
-    dropout.cpp
-    conv2d.cpp
-    pooling3d.cpp
-    soft_max.cpp
-    lrn_test.cpp
-    )
+file(GLOB TESTS *.cpp)
 
-set( SHORT_TESTS
-    immed_conv2d.cpp
-    immed_conv3d.cpp
-    activation.cpp
-    conv3d.cpp
-    bn_spatial_test.cpp
-    bn_peract_test.cpp
-    cba_inference.cpp
-    tensor_trans.cpp
-    na_train.cpp
-    w_supertensor.cpp
-    sqlite_perfdb.cpp
-    bn_3d_peract_test.cpp
-    tensor_transform.cpp
-    na_inference.cpp
-    bn_3d_spatial_test.cpp
-    ctc.cpp
-    rnn_vanilla.cpp
-    conv2d_bias.cpp
-    find_db.cpp
-    rnn_vanilla_dropout.cpp
-    tensor_cast.cpp
-    lstm.cpp
-    gru.cpp
-    main.cpp
-    lstm_dropout.cpp
-    gru_dropout.cpp
-    tensor_ops.cpp
-    handle_test.cpp
-    tensor_copy.cpp
-    tensor_set.cpp
-    tensor_scale.cpp
-    perfdb.cpp
-    conv3d_bias.cpp
-    cbna_inference.cpp
-    check_numerics_test.cpp
-    bn_aux.cpp
-    custom_allocator.cpp
-    mdgraph.cpp
-    fusion_aux.cpp
-    cache.cpp
-    type_name.cpp
-    test_errors.cpp
-    tensor_vec.cpp
-    tensor_test.cpp
-    solver.cpp
-    sequences.cpp
-    kernel_build_params.cpp
-    include_inliner.cpp
-    )
-
-foreach(TEST ${LONG_TESTS})
```

Comment:
What is the default value of COST? Shall we set is by default for all tests?

Reply:
Its zero.